var/spack/repos/builtin/packages/swipl/package.py

```diff
@@ -32,10 +33,14 @@ class Swipl(CMakePackage):
     variant('zlib', default=True, description='Compressed streams support')
     variant('odbc', default=True, description='ODBC database access')
     variant('unwind', default=True, description='Build with stack traces in crash reports')
+    variant('man', default=True, description='Install the HTML documentation')
```

Comment:
Does this build man pages and HTML documentation?

Reply:
I am not 100% sure. Quote form the CMakeLists.txt:

option(INSTALL_DOCUMENTATION
       "Install the HTML documentation files"
       ON)

Should I rename the option to remove potential confusion?

Reply:
Maybe rename `man` to `html` in that case?

Reply:
I dug a bit deeper into the build code. The man pages are installed unconditionally. I think it would be better to rename the flag then.